src/Agent.Plugins/PipelineArtifact/PipelineArtifactPluginV2.cs

```diff
@@ -37,11 +37,11 @@ public Task RunAsync(AgentTaskPluginExecutionContext context, CancellationToken
         // Properties set by tasks
         protected static class ArtifactEventProperties
         {
-            public static readonly string BuildType = "buildType";
+            public static readonly string PipelineType = "pipelineType";
             public static readonly string Project = "project";
             public static readonly string PipelineDefinition = "pipeline";
-            public static readonly string BuildTriggering = "specificBuildWithTriggering";
-            public static readonly string BuildVersionToDownload = "buildVersionToDownload";
+            public static readonly string PipelineTriggering = "specificPipelineWithTriggering";
```

Comment:
I don't have a great suggestion for `specificBuildWithTriggering` ... my best is `preferTriggeringPipeline`. It's a lot longer than everything else, but I'm banking on this being more of an advanced option.